lib/unit_conversion/distance_conversion.rb

```diff
@@ -1,44 +0,0 @@
-module DistanceConversion
```

Comment:
Is distance the right name for this file?  Should we use length, another unit name?